ALVRClient/Renderer.swift

```diff
@@ -44,123 +44,98 @@ let fullscreenQuadVertices:[Float] = [-panel_depth, -panel_depth, -panel_depth,
                                        0.5, 0]
 
 class Renderer {
-
+    
```

Comment:
If possible, please revert the whitespace changes in this file since they just pollute the diff.